### ISSUE TITLE
Feature flags growth work

### ISSUE THREAD

**Issue by liyiy** [PostHog/posthog#14257]
## Is your feature request related to a problem?

Feature flags has generally not been prioritized for growth and marketing work before and there are some opportunities to improve exposure here for that. 

## Describe the solution you'd like

Make PostHog feature flags become an entree instead of a snacc 🥺 

- [ ] Add feature flags/experiments to quick start guide
- [ ] Expose feature flags during onboarding?
- [ ] Marketing related work - maybe @joethreepwood can help? 😃 

## Additional context

Closely connected with this [feature flags implementation support](https://github.com/PostHog/posthog/issues/14219) work

**Comment by liyiy** [comment]
CC growth team @raquelmsmith @kappa90 for any ideas or suggestions too here 😄

**Comment by joethreepwood** [comment]
Non-leading devil's advocate question: Why the urgency to push FFs over, for example, other features which are paid for? Basically: would we not be better off pushing session recording? 

I don't actually have a strong feeling here, but I do want to make sure we're pushing the right things to users at the right time. 

Regarding onboarding, we already have this email in the flow that goes to all new users. 

<img width="1266" alt="Screenshot 2023-02-16 at 09 42 54" src="https://user-images.githubusercontent.com/84011561/219328560-71487ca3-b89d-4a9d-985e-1a5fdec263a8.png">

I can think of a few general marketing tactics we could do to push FFs, which I'll throw out there quickly and you can give me your thoughts?

- Push the LaunchDarkly integration live and collab with them on some content about the value of FFs
- Run an office hours session with Team Experimentation, inviting users to learn about how we use FFs
- Launch a series of blog articles about FFs ('Five Ways To Use FFs', etc) and adapt these into video
- I'd love to get some Twitter threads from Em and Raquel showing how our Growth Team uses FFs, which we could also adapt to content

I've also arranged for Luke to do a webinar in partnership with Engage.so early next month, about how to use and harness product data. We could push to make FFs and experiments a key part of that, potentially.

**Comment by neilkakkar** [comment]
Oh, very cool! @joethreepwood where does the 'find out more' button lead to?

**Comment by neilkakkar** [comment]
FFFF: Fast and furious feature flags 😂 

FFFF: Final fantasy feature flags 😂 

Damn, can I be marketing head already? 😂

**Comment by raquelmsmith** [comment]
I fully agree that Feature Flags have a lot of potential but need to be advertised better. At our offsite this week we discussed some ideas around this. I think the most relevant one is that each product should get its own onboarding - something like:

* a well-timed introduction to its availability
* a marketing-type page before you enter the product
* some very lightweight and well-timed prompts to get people to understand where things are and what they can do

This isn't something we are likely to get to in the next couple of weeks, but we are keen on it.

> Why the urgency to push FFs over, for example, other features which are paid for? Basically: would we not be better off pushing session recording?

Agreed that you only have so much you can tell people about and you have to be very selective. I think from my perspective there are two things here:

1. I _think_ the eventual goal is to turn Feature Flags and Experiments into their own paid product(s?). There are many companies that focus their entire offering around just these things. 
2. What we tell people about should be related to the things they are interested in. We've discussed having people tell us more about what they are interested in during signup or onboarding - analytics, feature flags, session recordings, etc. With this info we should be able to tailor the things we market to them based on their interests/needs (though you @joethreepwood will know more about our capabilities here than I would!)

> I'd love to get some Twitter threads from Em and Raquel showing how our Growth Team uses FFs, which we could also adapt to content

I haven't been active on twitter much lately but I could maybe swing something...

**Comment by joethreepwood** [comment]
> Oh, very cool! @joethreepwood where does the 'find out more' button lead to?

It directs to https://posthog.com/tutorials/categories/feature-flags

We direct here because, for SH users, we can't point to the app itself.

You have given me an idea to split this into two onboarding campaigns though. One for SH and one for Cloud. The cloud ones can direct to the app instead. 🤔 

(Adds to sprint)

**Comment by joethreepwood** [comment]
> Everything Raquel said

Fully agree with all this and would _love_ to team up with Growth again on these ideas. 

> 1. I _think_ the eventual goal is to turn Feature Flags and Experiments into their own paid product(s?). There are many companies that focus their entire offering around just these things.

This is definitely the plan, but as that isn't the state of play yet (whereas it is for session recording) I do think we can drive the biggest improvement to the bottom line with SR at the moment. 

> 2. What we tell people about should be related to the things they are interested in. We've discussed having people tell us more about what they are interested in during signup or onboarding - analytics, feature flags, session recordings, etc. With this info we should be able to tailor the things we market to them based on their interests/needs (though you @joethreepwood will know more about our capabilities here than I would!)

Our ability to market to people based on their preferences is very doable, provided we capture that information in a way which is passed to (mainly, ideally) Customer.io. Ideally we'd actual trigger everything as either a property or event, so that we can tailor onboarding in the app and in email using it. 

The question I'd have (and which may be worth an RFC) is what are the right things to target based on. For example, should we tailor onboarding messages based on user activity within the app, job title, size of company, industry, stated need, etc? I think we'd be well advised to choose a particular way to target and focus on that, as I've seen many onboarding systems get complicated by over-extending in this way.

**Comment by liyiy** [comment]
This is all great context folks! Thanks for the responses 😃
